examples/jsm/postprocessing/Pass.js

```diff
@@ -31,6 +31,8 @@ class Pass {
 
 	}
 
+	dispose() {}
```

Comment:
When this PR is merged, I'm planning to add more `dispose()` methods to the pass classes in `jsm/postprocessing` 👍 .